view.rb

```diff
@@ -1,6 +1,30 @@
 class View
-  def initialize()
 
+  def welcome
+    print "Welcome to Flashcards! Once you see the definition, enter your guess or quit to end the game.\n"
+    print '*' * 83
```

Comment:
@lksevans12 Hey Laura, can we have just a "puts" after this line so that the question prompt comes after the row of stars?